### PR TITLE
new comment

### PR DESCRIPTION
**Pull request by rebornix**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -18,7 +18,8 @@ hbs.registerPartials(__dirname + '/views/partials');
 app.set('view engine', 'hbs');
 
 // uncomment after placing your favicon in /public
```
Comment (rebornix) [inline app.js]: start comment

File: README.md [hunk 2]
```diff
@@ -1 +1 @@
-# pullrequest-demo
```
Comment (rebornix) [inline README.md]: start review

Comment (rebornix) [reply README.md]: start

Comment (rebornix) [reply README.md]: adsaf

File: app.js [hunk 3]
```diff
@@ -18,7 +18,8 @@ hbs.registerPartials(__dirname + '/views/partials');
 app.set('view engine', 'hbs');
```
Comment (RMacfarlane) [inline app.js]: what up

**Pull request closed without merging**